appengine/ndb/cache/snippets.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def set_in_process_cache_policy(ctx, func):
+    ctx.set_cache_policy(func)
```

Comment:
I agree that demonstrating an example func is good - but the flow of the doc page uses these snippets to build up to that. ie it first just shows an invocation with the 'func' placeholder. Then it describes what the func is, and then shows an example of it.